app/src/main/java/app/musikus/ui/theme/Theme.kt

```diff
@@ -74,43 +79,142 @@ fun MusikusTheme(
     }
 
     // enable Edge-to-Edge drawing with new API
-    val context = LocalContext.current as ComponentActivity
-    LaunchedEffect(key1 = theme) {
-
-        val statusBarColorDarkTheme = android.graphics.Color.TRANSPARENT
-        val statusBarColorLightTheme = android.graphics.Color.TRANSPARENT
-        val statusBarColorLightThemeOldDevices = DarkColorScheme.surface.toArgb()
-
-        val navBarColorDarkTheme = android.graphics.Color.TRANSPARENT
-        val navBarColorLightTheme = android.graphics.Color.TRANSPARENT
-        val navBarColorLightThemeOldDevices = DarkColorScheme.surface.toArgb()
-
-        context.enableEdgeToEdge(
-            statusBarStyle = if (useDarkTheme)
-                SystemBarStyle.dark(
-                    scrim = statusBarColorDarkTheme
-                )
+    // skip this block when in LocalInspectionMode (Compose Preview) so that we can use
+    // MusikusTheme{} wrapper also in Compose Previews
+    if (!LocalInspectionMode.current) {
+        val context = LocalContext.current as ComponentActivity
+        LaunchedEffect(key1 = theme) {
+
+            val statusBarColorDarkTheme = android.graphics.Color.TRANSPARENT
+            val statusBarColorLightTheme = android.graphics.Color.TRANSPARENT
+            val statusBarColorLightThemeOldDevices = DarkColorScheme.surface.toArgb()
+
+            val navBarColorDarkTheme = android.graphics.Color.TRANSPARENT
+            val navBarColorLightTheme = android.graphics.Color.TRANSPARENT
+            val navBarColorLightThemeOldDevices = DarkColorScheme.surface.toArgb()
+
+            context.enableEdgeToEdge(
+                statusBarStyle = if (useDarkTheme)
+                    SystemBarStyle.dark(
+                        scrim = statusBarColorDarkTheme
+                    )
                 else SystemBarStyle.light(
                     scrim = statusBarColorLightTheme,
                     darkScrim = statusBarColorLightThemeOldDevices
                 ),
-            navigationBarStyle = if (useDarkTheme)
-                SystemBarStyle.dark(
-                    scrim = navBarColorDarkTheme
-                )
+                navigationBarStyle = if (useDarkTheme)
+                    SystemBarStyle.dark(
+                        scrim = navBarColorDarkTheme
+                    )
                 else SystemBarStyle.light(
                     scrim = navBarColorLightTheme,
                     darkScrim = navBarColorLightThemeOldDevices
                 ),
-        )
+            )
+        }
     }
-
```

Comment:
Why remove this empy line? I think it adds a nice separation between the conditional block above and the main content of the whole `MusikusTheme()`